grunttasks/concat.js

```diff
@@ -7,6 +7,10 @@ module.exports = function (grunt) {
     requirejs: {
       dest: '<%= yeoman.tmp %>/scripts/main.js',
       src: ['app/bower_components/requirejs/require.js', '<%= yeoman.tmp %>/scripts/main.js']
+    },
+    easteregg: {
```

Comment:
this is the wrong way of doing that.

maybe create an easter egg module in `scripts/lib` and have it included in app-start or something. also only show it in production environments (see if you can find a way to do that, if not we are probably going to be adding that to /config soon)